compose-common.yml

```diff
@@ -12,35 +12,49 @@ base:
     # --link some-rabbit:rabbit "just works"
     - DJANGO_CELERY_BROKER_URL=amqp://guest@rabbit
     - DJANGO_DATABASE_URL=postgis://postgres@database/postgres
-    - DJANGO_EMAIL_HOST=localhost
-    # Settings above likely need to be changed, depending on  the environment
+    - APP_PORT=8000
+    - APP_HOST=0.0.0.0
+
+    # Logging
+    - PYTHONUNBUFFERED=non-empty-string
```

Comment:
I guessed it might be something like that. I guess Pythons environment interface is to blame here. It'd be easier to understand if it'd either check for strings representing booleans, like `y`, `yes`, `Off`, etc., or just react on the variable being set or not.

Using this quirk to make the value self-documenting (and documenting the quirk) is a good idea, though. How about

        - PYTHONUNBUFFERED=unbuffered because this string is not empty - see https://docs.python.org/3/using/cmdline.html#envvar-PYTHONUNBUFFERED